modules/client/src/client-common/utils.ts

```diff
@@ -35,19 +35,21 @@ export function computeProposalStatus(
     parseInt(proposal.startDate) * 1000,
   );
   const endDate = new Date(parseInt(proposal.endDate) * 1000);
+  if (proposal.executed) {
+    return ProposalStatus.EXECUTED;
+  }
   if (startDate >= now) {
     return ProposalStatus.PENDING;
-  } else if (proposal.executed) {
-    return ProposalStatus.EXECUTED;
-  } else if (endDate >= now) {
-    return ProposalStatus.ACTIVE;
-  } else if (
-    proposal.executable
+  }
+  if (
+    proposal.executable || proposal.earlyExecutable
   ) {
     return ProposalStatus.SUCCEEDED;
-  } else {
-    return ProposalStatus.DEFEATED;
   }
+  if (endDate >= now) {
```

Comment:
Very cosmetic, but grouping by `else if()` shows related branches as part of the same thing

Reply:
they are not related. if one matches the function returns and aborts. if not the code flow continues